new/components/utils

```diff
@@ -128,6 +107,21 @@ function download_file
     echo ${destination_file}
 }
 
+function download_cloudify_resource() {
+    url=$1
+    destination_file="/etc/cloudify/$(get_file_name $url)"
```

Comment:
add variable DOWNLOAD_PREFIX or something similar instead of hardcoding this, will be easier to change